src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorSearchIntegrationTest.java

```diff
@@ -405,6 +403,178 @@ public void insertInvalidVectorData() {
           .body("errors[0].errorCode", is("SHRED_BAD_VECTOR_VALUE"))
           .body("errors[0].exceptionClass", is("JsonApiException"));
     }
+
+    @Test
+    public void insertSimpleBinaryVector() {
+      final String id = UUID.randomUUID().toString();
+      final float[] expectedVector = new float[] {0.25f, 0.25f, 0.25f, 0.25f, 0.25f};
```

Comment:
I forgot to suggest but let's use vector with some variation in values, so not all same.

Like 

```
{ 0.25, -1.5, 0.0, .... }
```

or whatever.